src/Discord/Voice/VoiceClient.php

```diff
@@ -1100,18 +1119,27 @@ public function unpause(): void
 
     /**
      * Stops the current sound.
+     *
+     * @return ExtendedPromiseInterface
+     * @throws \Exception
      */
-    public function stop(): void
+    public function stop(): ExtendedPromiseInterface
```

Comment:
Are you sure this wont allow the code being repeated?
```php
$vc->stop();
$vc->stop();
```

Reply:
it wouldn't matter if you repeat it, the second would return the exception that the audio is no longer playing